packages/table/src/spectrum-table-row.css

```diff
@@ -0,0 +1,171 @@
+:host([dir='ltr'][drop-target]):before {
+    left: 0; /* [dir=ltr] .spectrum-Table-body.is-drop-target:before,
+   * [dir=ltr] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+:host([dir='rtl'][drop-target]):before {
+    right: 0; /* [dir=rtl] .spectrum-Table-body.is-drop-target:before,
+   * [dir=rtl] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+:host([dir='ltr'][drop-target]):before {
+    right: 0; /* [dir=ltr] .spectrum-Table-body.is-drop-target:before,
+   * [dir=ltr] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+:host([dir='rtl'][drop-target]):before {
+    left: 0; /* [dir=rtl] .spectrum-Table-body.is-drop-target:before,
+   * [dir=rtl] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+:host([drop-target]):before {
+    bottom: 0;
+    content: ''; /* .spectrum-Table-body.is-drop-target:before,
+   * .spectrum-Table-row.is-drop-target::before,
+   * .spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * .spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * .spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * .spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+    position: absolute;
+    top: 0;
+    z-index: 1;
+}
+:host {
+    cursor: pointer;
+    position: relative; /* .spectrum-Table-row */
+    transition: background-color
+        var(--spectrum-global-animation-duration-100, 0.13s) ease-in-out;
+}
+:host(:focus) {
+    outline: 0; /* .spectrum-Table-row:focus */
+}
+:host([dir='ltr']) .spectrum-Table-checkboxCell {
```

Comment:
Room to remove some more selectors in `spectrum-config.json` or maybe correct them, as this might support delivering the Checkbox Cell better...